src/app/components/header/header.component.html

```diff
@@ -1,6 +1,7 @@
 <nav>
   <div class="left-side-links">
-    <a class="link" [routerLink]="authentication.getUrl"> home </a>
+    <a class="link" [routerLink]="['home/public']"> home </a>
+    <a *ngIf="authentication.userValue"  class="link" [routerLink]="authentication.getUrl"> My acount </a>
     <a class="link" href="about"> about us </a>
   </div>
   <div class="right-side-links">
```

Comment:
that navigates* 
other than that this commit is amazing, you have set up the router link so it points to the correct portal
GOOD JOB

Reply:
also at the end of line 4 its my account*